extensions/wikia/SEOTweaks/scripts/uncrawlable-urls.js

```diff
@@ -6,14 +6,19 @@ require([
 	window
 ) {
 	'use strict';
-	$(function () {
+
+	const decodeUncrawlableURL = function() {
 
 		// Handles middle click, ctrl+click and regular click
 		$('a[data-uncrawlable-url]').on('mousedown', function () {
 			var $this = $(this);
 			var url = window.atob($this.attr('data-uncrawlable-url'));
 			$this.attr('href', url);
 		});
-	});
+	};
+
+	$(decodeUncrawlableURL);
+
+	$( window ).on( 'EditPageAfterRenderPreview', decodeUncrawlableURL);
```

Comment:
Way more content-displaying asynchronous code is using the `wikipage.content` hook. This only covers edit previews, while `wikipage.content` is used (besides being fired for edit previews) on, for example, setting hooks on new posted threads:
https://github.com/Wikia/app/blob/1e4ea22073c29ec97beeccda86d068915366d0c5/extensions/wikia/Wall/js/WallMessageForm.js#L71
It is also used a lot in userscripts, for example [AddRailModule](https://dev.wikia.com/wiki/MediaWiki:AddRailModule/code.js) (so if the `wikipage.content` hook isn't used here, user-created Rail modules will contain redlinks that don't work. Clearly, this isn't a SEO concern, but since the issue affects more than just edit previews this easy change should fix it for more than just them).